src/parser/jobs/sch/ClericStance.js

```diff
@@ -0,0 +1,193 @@
+import React, {Fragment} from 'react'
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS, {getAction} from 'data/ACTIONS'
+import Module from 'parser/core/Module'
+import {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import {Accordion, Message, Icon} from 'semantic-ui-react'
+import Rotation from 'components/ui/Rotation'
+import STATUSES from 'data/STATUSES'
+
+// all these suggestions for CS are ultimately trivial :notlikeblob:
+const defaultSeverityTiers = {
+	1: SEVERITY.MINOR,
+}
+
+// list of things to track for CS usage.
+// can override the suggestions from the default template here
+const EXPECTED_CASTS = [
```

Comment:
Try formatting these a little cleaner and consistent, so they're easier to read.
Along the lines of

```javascript
{
    ...getAction(ACTIONS.CELRIC_STANCE.id),
    name: 'GCD',
    count: 6,
}
```

is probably better, since this will, especially when there is big properties like `content` make keys easier to find.